recipes/basemap-data-hires/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+    name: basemap-data-hires
+    version: 1.0.7
+
+source:
+  git_url: https://github.com/matplotlib/basemap.git
+  git_tag: v1.0.7rel
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - basemap
+    run:
+        - basemap
+
+test:
+    imports:
+        - mpl_toolkits.basemap
```

Comment:
I added this to escape the linter. The real test is a plot in the `run_test.py` that uses the high resolution data. I'd like to keep that to ensure that the patch works and basemap finds the data.